Shared/View/AuthenticationView.swift

```diff
@@ -16,6 +16,8 @@ struct AuthenticationView: View {
     }
 }
 
+// test123@gmail.com
+// 123123
```

Comment:
I think these can go to our Wiki https://github.com/Fun-Gen/Fun-Gen/wiki where we include information about test accounts, and maybe link to the Wiki page here